packages/e2e/.detoxrc.json

```diff
@@ -0,0 +1,57 @@
+{
+  "skipLegacyWorkersInjection": true,
+  "apps": {
+    "ios.debug": {
```

Comment:
Curious why we need both `debug` and `release` configs for both platforms?

Reply:
What do you think about`dev` and `ci` to be explicit?

Reply:
This config is specific to ios/android apps build which can be either build or release so prefer to keep nomenclature. It's the Detox recommendation as well (https://wix.github.io/Detox/docs/config/overview#config-structure).

Reply:
Thanks for the context!